Sources/SmokeDynamoDB/AWSDynamoDBCompositePrimaryKeyTable+updateItems.swift

```diff
@@ -203,6 +203,44 @@ public extension AWSDynamoDBCompositePrimaryKeyTable {
         return EventLoopFuture.andAllSucceed(futures, on: self.eventLoop)
     }
     
+    func monomorphicBulkWriteWithFallback<AttributesType, ItemType>(_ entries: [WriteEntry<AttributesType, ItemType>]) -> EventLoopFuture<Void> {
+        // fall back to singel operation if the write entry exceed the statement length limitation
```

Comment:
Typo `singel`
"if the write entry exceed**s**"
Same at L529, InMemoryDynamoDBCompositePrimaryKeyTable.swift#L138, InMemoryDynamoDBCompositePrimaryKeyTableWithIndex.swift#L112 and SimulateConcurrencyDynamoDBCompositePrimaryKeyTable.swift#L138.

Reply:
Fixed